packages/databricks-vscode/package.json

```
@@ -444,17 +451,22 @@
             },
             {
                 "view": "configurationView",
-                "contents": "There are multiple Databricks projects in the folder:\n[Open existing Databricks Project](command:databricks.bundle.openSubProject)",
+                "contents": "Detected multiple Databricks projects in the VSCode workspace:\n[Select a project](command:databricks.bundle.selectActiveProjectFolder)",
```

```suggestion
                "contents": "Multiple Databricks projects were detected in the workspace:\n[Choose a project](command:databricks.bundle.selectActiveProjectFolder)",
```
It's a fuzzy (and honestly not super important) line, but it seems like this is a choice, not just a selection.

And poking around, I'm not sure VSCode is approved product terminology, so instead of expanding it, I'd avoid it by removing it.